src/west/app/project.py

```diff
@@ -212,20 +214,22 @@ def local(self, args) -> Path:
             log.die('--mr cannot be used with -l')
 
         manifest_dir = Path(args.directory or os.getcwd())
-        manifest_file = manifest_dir / 'west.yml'
+        manifest_filename = args.manifest_file or 'west.yml'
+        manifest_file = manifest_dir / manifest_filename
         topdir = manifest_dir.parent
         rel_manifest = manifest_dir.name
         west_dir = topdir / WEST_DIR
 
         if not manifest_file.is_file():
-            log.die(f'can\'t init: no "west.yml" found in {manifest_dir}')
+            log.die(f'can\'t init: no manifest found in {manifest_dir}')
```

Comment:
I think it would be better to say exactly the file that's being searched for:

```suggestion
            log.die(f'can\'t init: no {manifest_filename} found in {manifest_dir}')
```